metabox/metabox/core/lxd_provider.py

```diff
@@ -125,7 +133,11 @@ def _create_profiles(self):
                 logger.debug(
                     '{} LXD profile created successfully', profile_name)
 
-    def _create_machine(self, config):
+    def _create_machine(self, config, use_existing=False):
+        assert (
+            not use_existing or
+            (use_existing and config.origin == 'source')
```

Comment:
This cannot be asserted, therefore don't use assert. Assert is for when you control all of the axioms.
This, speaking very broadly, is a "contradicting input" kind of input error.